src/report/js/ReportBuilder.js

```diff
@@ -3786,8 +3809,10 @@ define([
                       } else {
                         return null;
                       }
```

Comment:
Some of the keys already include 'Alerts', so we need an extra `else if` case. If it doesn't have "Fire alerts" already, we format it to add the text. Otherwise, we just return the key.